File: UsbSerialConsole/app/src/main/java/jp/sugnakys/usbserialconsole/settings/LicenseDialogFragment.java
package jp.sugnakys.usbserialconsole.settings;

import android.annotation.SuppressLint;
import android.app.AlertDialog;
import android.app.Dialog;
import android.app.DialogFragment;
import android.content.Context;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.webkit.WebView;
import android.webkit.WebViewClient;

import jp.sugnakys.usbserialconsole.R;

public class LicenseDialogFragment extends DialogFragment {

    @Override
    public Dialog onCreateDialog(Bundle savedInstanceState) {
        AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());

        LayoutInflater inflater = (LayoutInflater)getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        @SuppressLint("InflateParams") View content = inflater.inflate(R.layout.license_view_main, null);

        WebView webView = (WebView)content.findViewById(R.id.webview);
        webView.setWebViewClient(new WebViewClient());
        webView.loadUrl("file:///android_asset/license/license.html");

        builder.setTitle(getString(R.string.license_title));
        builder.setView(content);

        return builder.create();
    }
}

File: UsbSerialConsole/app/src/main/java/jp/sugnakys/usbserialconsole/settings/LicenseDialogFragment.kt
package jp.sugnakys.usbserialconsole.settings

import android.os.Bundle
import android.view.LayoutInflater
import android.annotation.SuppressLint
import android.app.AlertDialog
import android.app.Dialog
import android.content.Context
import android.view.View
import android.webkit.WebView
import jp.sugnakys.usbserialconsole.R
import android.webkit.WebViewClient
import androidx.fragment.app.DialogFragment

class LicenseDialogFragment : DialogFragment() {
    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
        val builder = AlertDialog.Builder(activity)
        val inflater = activity?.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
        @SuppressLint("InflateParams") val content =
            inflater.inflate(R.layout.fragment_license, null)
        val webView = content.findViewById<View>(R.id.webview) as WebView
        webView.webViewClient = WebViewClient()
        webView.loadUrl("file:///android_asset/license/license.html")
        builder.setTitle(getString(R.string.license_title))
        builder.setView(content)
        return builder.create()
    }
}